geopandas/geoseries.py

```diff
@@ -594,7 +594,30 @@ def _constructor(self):
     def _constructor_expanddim(self):
         from geopandas import GeoDataFrame
 
-        return GeoDataFrame
+        def expanddim(data=None, index=None, crs=None, **kwargs):
+            # if geometry is somehow supplied, ignore it because we don't need it
+            kwargs.pop("geometry", None)
+            if isinstance(data, GeoSeries):
+                # pandas default column name is 0, keep convention
+                geo_col_name = data.name if data.name is not None else 0
+            elif isinstance(data, dict):  # Dict of {name : (Geo)Series}
+                # if only one key, this is not ambiguous, if more than one
+                # i.e. from pd.concat([GeoSeries, GeoSeries]) the geom col comes
+                # from the left (same as for multiple gdfs in concat)
+                geo_col_name = list(data.keys())[0]
+            else:  # blockmanager
+                geo_col_name = data.axes[0][0]
+
```

Comment:
I think it's fine to know / assume this is a block manager. But I am only wondering a bit if the `data.axes[0][0]` isn't too much assumption on that it will be a blockmanager with only a single column.

Reply:
This is trying to get the "leftmost column name", to be consistent with the dict case above. I'm not very familiar with pandas internals (have learnt a lot over this series of PRs but still not much) and I haven't come across a great reference for them. There was more brute force inspecting blockmanager attributes in a debugger than I'd like to admit there, so it's entirely plausible that's not suitable/ there's a better solution.